nextcord/user.py

```diff
@@ -247,11 +247,14 @@ def created_at(self) -> datetime:
     def display_name(self) -> str:
         """:class:`str`: Returns the user's display name.
 
-        For regular users this is just their username, but
-        if they have a guild specific nickname then that
-        is returned instead.
+        Returns the users displayed name.
+        So whatever is the first thing to exist from the following priority list:
+
+        1. Guild specific nickname
```

Comment:
This doesn't apply since this is for `User`, so you probably want to just mention that it returns the global name, or username